data/VehicleConfig/vehicle.json

```diff
@@ -0,0 +1,54 @@
+{
+	"Vehicles": [{
+			"name": "v1",
```

Comment:
why do we need id, speed and bearing. these are dynamic things which would be decided in simulation.